src/Microsoft.Diagnostics.Runtime/src/Builders/RuntimeBuilder.cs

```diff
@@ -1570,7 +1568,11 @@ private static bool IsArraySubstring(string name, int start, int end)
         [return: NotNullIfNotNull("name")]
         public static string? FixGenerics(string? name)
         {
-            if (name == null || name.IndexOf("[[") == -1)
+#if NETCOREAPP3_1
```

Comment:
Instead of putting #if statements in the code, we have been solving this via adding extension methods on platforms where functions are missing.  For example:  https://github.com/microsoft/clrmd/blob/master/src/Microsoft.Diagnostics.Runtime/src/Extensions/SpanExtensions.cs

Would you mind changing this to just be `name.Contains` for both cases, and add an extension method for non-NETCOREAPP3_1?